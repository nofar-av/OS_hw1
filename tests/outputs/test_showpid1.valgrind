==28610== Memcheck, a memory error detector
==28610== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==28610== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==28610== Command: /home/omer/OS_wet1/smash
==28610== Parent PID: 28608
==28610== 
==28610== 
==28610== FILE DESCRIPTORS: 4 open at exit.
==28610== Open file descriptor 3: /home/omer/OS_wet1/tests/outputs/test_showpid1.valgrind
==28610==    <inherited from parent>
==28610== 
==28610== Open file descriptor 2: /home/omer/OS_wet1/tests/outputs/test_showpid1.err
==28610==    <inherited from parent>
==28610== 
==28610== Open file descriptor 1: /home/omer/OS_wet1/tests/outputs/test_showpid1.out
==28610==    <inherited from parent>
==28610== 
==28610== Open file descriptor 0:
==28610==    <inherited from parent>
==28610== 
==28610== 
==28610== HEAP SUMMARY:
==28610==     in use at exit: 0 bytes in 0 blocks
==28610==   total heap usage: 13 allocs, 13 frees, 81,384 bytes allocated
==28610== 
==28610== All heap blocks were freed -- no leaks are possible
==28610== 
==28610== For counts of detected and suppressed errors, rerun with: -v
==28610== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
