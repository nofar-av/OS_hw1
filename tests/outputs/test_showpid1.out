smash> smash pid is 6000
smash> 